sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosClientEncryptionKeyResponse.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.ClientEncryptionKey;
+import com.azure.cosmos.implementation.ResourceResponse;
+import com.azure.cosmos.implementation.apachecommons.lang.StringUtils;
+
+/**
+ * The type Cosmos client encryption key response.
+ */
+public class CosmosClientEncryptionKeyResponse extends CosmosResponse<CosmosClientEncryptionKeyProperties>  {
```

Comment:
this being in the azure-cosmos will create confusion for customer who are not intending to use Encryption as Intellj AutoComplete will show these when they type "cosmos"

Reply:
When/If we move CosmosClientEncryptionKeyProperties/CosmosClientPolicy/ClientEncryptionIncludedPath into encryption project and start using ObjectNode in Cosmos project then we can move CosmosClientEncryptionKeyResponse to encryption as well. Before that we cannot. Added beta anotation on its api.

Reply:
Resolving as per  after public api review